main.go

```diff
@@ -36,6 +35,7 @@ var (
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 	utilruntime.Must(deliveryv1alpha1.AddToScheme(scheme))
+	utilruntime.Must(v2beta1.AddToScheme(scheme))
```

Comment:
`autoscaling`? 🤔

Reply:
yep just spotted this. have the fix now.